src/tools/compiletest/src/common.rs

```diff
@@ -376,6 +376,7 @@ pub struct Config {
     pub only_modified: bool,
 
     pub target_cfgs: OnceLock<TargetCfgs>,
+    pub builtin_cfgs: OnceLock<Vec<String>>,
```

Comment:
Remark: I feel like this is kinda overkill, but I'll see if this is even needed during the rework